frontend/src/utils/format_column_data.ts

```
@@ -190,23 +276,43 @@ export class GraphData {
           );
 
           const [p_x, p_y] = prev_nodes[parent_idx];
-          const [x, y] = curr_nodes[written_nodes];
+          const [l_x, l_y] = curr_nodes[written_nodes];
 
-          if (!p_x || !p_y || !x || !y) {
-            throw new Error("Could not find parent: " + p_x + p_y + x + y);
+          if (!p_x || !p_y || !l_x || !l_y) {
+            throw new Error("Could not find parent: " + p_x + p_y + l_x + l_y);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)